extension/src/popup/components/sendPayment/SendTo/index.tsx

```diff
@@ -196,7 +196,12 @@ export const SendTo = ({ previous }: { previous: ROUTES }) => {
         title="Send To"
         customBackAction={() => navigateTo(previous)}
       />
-      <form>
+      <form
```

Comment:
fixing how pressing enter on the SendTo and SendAmount inputs submits the form by mistake
https://stellarorg.atlassian.net/browse/WAL-125?filter=10043